bot/toolkit-base/src/main/kotlin/BotIoc.kt

```diff
@@ -37,7 +38,7 @@ object BotIoc {
      * The core modules of the bot.
      */
     val coreModules: List<Module> =
-        listOf(sharedModule, botModule, botMongoModule, noOpTranslatorModule, noOpSTTModule)
+        listOf(sharedModule, botModule, botMongoModule, noOpTranslatorModule, noOpSTTModule, awsToolsModule)
```

Comment:
N'ajouter awsToolsModule que si il est dans le classpath

Reply:
Use a ServiceLoader:

- define a BotAdditionalModuleService interface in shared
- in aws-tool, add an implementation AwsToolBotAdditionalModuleService of this interface (with a META-INF with file)
- in botIoc, add:

```kotlin
val coreModules: List<Module> =
        listOf(sharedModule, botModule, botMongoModule, noOpTranslatorModule, noOpSTTModule) + loadServices<BotAdditionalModuleService>.map { it.module() }
```

Reply:
All has been done